packages/graph/workers/src/forceDirectedOptions.ts

```diff
@@ -0,0 +1,26 @@
+export interface Node {
+    id: string;
+    text: string;
+}
+
+export interface Link {
+    id: string;
+    source: Node;
+    target: Node;
+}
+
+export interface Data {
+    nodes: Node[];
+    links: Link[];
+}
+
+export interface Options {
+    alpha: number;
+    alphaMin: number;
+    alphaDecay: number;
+    velocityDecay: number;
+    repulsionStrength: number;
+    linkDistance: number;
+    linkStrength: number;
+    iterations: number;
+}
```

Comment:
There are situations where we might need some of these variables to be set on a per-vertex or per-edge basis rather than a graph-wide constant. For example:
An Edge representing DataPatterns' negative numeric correlations should have a larger linkDistance